relay-event-normalization/src/normalize/span/country_subregion.rs

```diff
@@ -0,0 +1,261 @@
+// country_subregion.rs
+use std::collections::HashMap;
+use std::fmt;
+
+/** Subregions regions as defined by UN M49 standards, the enum value corresponds to the region code */
+pub enum Subregion {
+    NorthernAmerica = 021,
+    CentralAmerica = 013,
+    Caribbean = 029,
+    SouthAmerica = 005,
+    NorthernEurope = 154,
+    WesternEurope = 155,
+    SouthernEurope = 039,
+    EasternEurope = 151,
+    EasternAsia = 030,
+    SouthernAsia = 034,
+    SouthEasternAsia = 035,
+    WesternAsia = 145,
+    CentralAsia = 143,
+    NorthernAfrica = 7,
+    WesternAfrica = 011,
+    MiddleAfrica = 017,
+    EasternAfrica = 014,
+    SouthernAfrica = 018,
+    Melanesia = 054,
+    Micronesia = 057,
+    Polynesia = 061,
+    AustraliaAndNewZealand = 053,
+}
+
+impl fmt::Display for Subregion {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        let subregion_str = match self {
+            Subregion::NorthernAmerica => "Northern America",
+            Subregion::SouthAmerica => "Southern America",
+            Subregion::CentralAmerica => "Central America",
+            Subregion::Caribbean => "Caribbean",
+            Subregion::NorthernAfrica => "Northern Africa",
+            Subregion::SouthernEurope => "Southern Europe",
+            Subregion::WesternAsia => "Western Asia",
+            Subregion::EasternEurope => "Eastern Europe",
+            Subregion::EasternAsia => "Eastern Asia",
+            Subregion::SouthEasternAsia => "South-Eastern Asia",
+            Subregion::CentralAsia => "Central Asia",
+            Subregion::SouthernAsia => "Southern Asia",
+            Subregion::WesternEurope => "Western Europe",
+            Subregion::NorthernEurope => "Northern Europe",
+            Subregion::MiddleAfrica => "Middle Africa",
+            Subregion::WesternAfrica => "Western Africa",
+            Subregion::EasternAfrica => "Eastern Africa",
+            Subregion::SouthernAfrica => "Southern Africa",
+            Subregion::Melanesia => "Melanesia",
+            Subregion::Micronesia => "Micronesia",
+            Subregion::Polynesia => "Polynesia",
+            Subregion::AustraliaAndNewZealand => "Australia and New Zealand",
+        };
+        write!(f, "{}", subregion_str)
+    }
+}
+
+/**
+ * Maps ISO 2 letter country codes to geoscheme regions based on the UN M.49 standard.
+ */
+pub fn get_country_to_subregion() -> HashMap<&'static str, Subregion> {
```

Comment:
Function should be passed the iso code and return the `Subregion`, implementation can use a single `match`.